pkg/apis/istio/v1alpha1/fixup_structs/main.go

```diff
@@ -111,11 +111,34 @@ func main() {
 		}
 	}
 
+	if strings.Contains(filePath, "values_types") {
+		out = patchValues(out)
+	}
 	if err := ioutil.WriteFile(filePath, []byte(strings.Join(out, "\n")), 0644); err != nil {
 		fmt.Println(err)
 		os.Exit(1)
 	}
+}
 
+// patchValues is helper function to patch specific fields of the generated values_types.pb.go
```

Comment:
could you expand on why this is needed? it will be useful to maintainers trying to understand why this was created.

Reply:
added more comments